scripts/generate-components-handlers/event-handlers-parser.ts

```diff
@@ -209,6 +210,7 @@ const getEventHandlersParser = (
     return interfaces.reduce((componentsMap, interfaceNode) => {
       const interfaceNodeName = interfaceNode.name.getText();
       return Object.assign(componentsMap, {
+        // you can use here EVENTS_MODULE_PREFIX
```

Comment:
I actually think it's more readable like this, so I removed `EVENTS_MODULE_PREFIX`